tests/device_tests/ethereum/test_signtx.py

```diff
@@ -209,14 +210,21 @@ def test_data_streaming(client: Client):
     """
     with client:
         is_t1 = client.model is models.T1B1
+        is_mercury = client.model is models.T3T1
         client.set_expected_responses(
             [
                 messages.ButtonRequest(code=messages.ButtonRequestType.SignTx),
                 (is_t1, messages.ButtonRequest(code=messages.ButtonRequestType.SignTx)),
                 (
-                    not is_t1,
+                    not is_t1 and not is_mercury,
                     messages.ButtonRequest(code=messages.ButtonRequestType.Other),
                 ),
+                (
```

Comment:
Why was it necessary to have a different `BR`s for `mercury` in this flow?

Reply:
It was not. Thank you for pointing it: https://github.com/trezor/trezor-firmware/pull/4095/commits/ea4a00a6582e943a7a4af1448999ae7f0800b31c